docs/useGetOne.md

```diff
@@ -90,7 +90,9 @@ const UserProfile = () => {
 
 This results in less calls to the dataProvider. For instance, if the `<UserProfile>` component above is rendered in a `<Datagrid>`, it will only make one call to `dataProvider.getMany()` for the entire list instead of one call to `dataProvider.getOne()` per row.
 
-As this hook is often used to fetch references, react-admin exposes a `useReference` hook, which avoids doing the array conversion manually. It's an application hook rather than a data provider hook, so its syntax is a bit different. Prefer `useReference` to `useGetManyAggregate` when you use `useGetOne` to fetch a reference.
+## `useReference`
+
+As `useGetManyAggregate` hook is often used to fetch references, react-admin exposes a `useReference` hook, which avoids doing the array conversion manually. It's an application hook rather than a data provider hook, so its syntax is a bit different. Prefer `useReference` to `useGetManyAggregate` when you use `useGetOne` to fetch a reference.
```

Comment:
I find it more confusing to split this into two sections, because both `useGetManyAggregate` and `useReference` aim at doing the same thing (only the cardinality differ), and both can fall under the category "Aggregating `getOne` Calls".
I found it clearer the way it was before.
@fzaninotto what do you think?

Reply:
agreed, I just modified it